up_esb/bridge.py

```diff
@@ -316,7 +316,7 @@ def set_initial_values(self, problem: Problem) -> None:
         for fluent in problem.fluents:
             for parameter in fluent.signature:
                 # Avoid redundancy.
-                if parameter.type not in type_objects.keys():
+                if parameter.type not in type_objects:
```

Comment:
Why fix things which are not broken? I know Python even suggests to omit the `.keys()` but personally I recommend to use interfaces as explicit as possible. Otherwise, especially in Python due to its quick and dirty nature, code can continue to run just fine until an error occurs later somewhere else. In this case, it's not critical because, so it's up to you.

Reply:
Actually, after writing my point of view, you probably received a warning or hint due to the checks, I presume?